static/js/publisher/release/components/releasesConfirm.js

```diff
@@ -23,13 +24,16 @@ class ReleasesConfirm extends Component {
   }
 
   onRevertClick() {
+    this.props.triggerGAEvent("click-revert");
```

Comment:
This can be done in the action

Reply:
Will keep this here to have both `click-revert` and `click-save` defined in the same file